rolemanager/module.py

```diff
@@ -0,0 +1,125 @@
+        """Create embed for member list.
+        Args:
+            ctx: Command context.
+            option: Item's title.
+            description: list of items.
+        Returns: :class:`discord.Embed` information embed
+        """
+        chunked_list = list()
+        chunk_size = 15
+
+        for i in range(0, len(description), chunk_size):
+
+            page = utils.discord.create_embed(
+                author=ctx.author,
+                title=title,
+                description="\n".join(description[i : i + chunk_size]),
+            )
+
+            chunked_list.append(description[i : i + chunk_size])
+
+            elements.append(page)
+
+        return elements
+
+    # MAIN
+    @commands.guild_only()
+    @commands.group(name="rolemanager")
+    @check.acl2(check.ACLevel.MOD)
+    async def rolemanager_(self, ctx):
+        """
+        preview [base_role] [role_to_remove]
+        execute [base_role] [role_to_remove]
+
+        """
+        await utils.discord.send_help(ctx)
+
+    @check.acl2(check.ACLevel.MOD)
+    @rolemanager_.command(name="preview")
+    async def rolemanager_preview(
+        self, ctx, role_base: discord.Role, *, role_remove: discord.Role
+    ):
+        """
+        List all users with base role and selected role to remove.
+        """
+        base_members = set(role_base.members)
+        remove_members = set(role_remove.members)
+        role_intersection = base_members & remove_members
+
+        if role_intersection is not None:
+
+            title = _(ctx, "Members with forbidden role")
+
+            member_list = list((str(member) for member in role_intersection))
+
+            embeds = RoleManager._create_embeds(
+                ctx=ctx,
+                title=title,
+                description=member_list,
+            )
+
+            scrollable_embed = ScrollableEmbed(ctx, embeds)
+            await scrollable_embed.scroll()
+
+        else:
+            await ctx.reply(_(ctx, "No member with this role"))
+
+    @check.acl2(check.ACLevel.MOD)
+    @rolemanager_.command(name="execute")
+    async def rolemanager_execute(
+        self, ctx, role_base: discord.Role, *, role_remove: discord.Role
+    ):
+        """
+        Execute command to remove selected role from members with base role.
+        """
+        base_members = set(role_base.members)
+        remove_members = set(role_remove.members)
+        role_intersection = base_members & remove_members
+
+        if role_intersection is not None:
+
+            embed = discord.Embed(
+                title=_(ctx, "REMOVE ROLE FROM MEMBERS"),
+                description=_(
+                    ctx,
+                    "Are you sure you want from users with role: {role_base} remove role: {role_remove}?",
+                ).format(role_base=role_base.mention, role_remove=role_remove.mention),
+            )
+
+            view = ConfirmView(ctx, embed)
+            value = await view.send()
+            if value is None:
```

Comment:
You can shorten this if/elif/else if you change it as:

```Python
if value:
    # do the removal
else:
   # send Aborted message
```

In the end it's not important for the user if the removal was aborted manually or by timeout.

Reply:
Done.

Reply:
Nope, it's still not commited

Reply:
Now committed.